README.md

```diff
@@ -48,6 +48,7 @@ console.log(
 * [CSS Modules](./packages/css-module/)
 * [deno 'npm:' prefix](./packages/deno-npm-prefix/)
 * [JSONC](./packages/jsonc/)
+* [JSON5](./packages/json5/)
```

Comment:
Alphabetical order please 🙂 (JSON5 should go before JSONC)

Reply:
thanks I edited